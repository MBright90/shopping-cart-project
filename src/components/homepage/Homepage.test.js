import { render } from '@testing-library/react'
import React from 'react'

import Homepage from './index'

jest.mock('@assets/images/background')

describe('HomepageImage', () => {
  it('renders to match snapshot', () => {
    const { container } = render(<Homepage />)
    expect(container).toMatchSnapshot()
  })
})
