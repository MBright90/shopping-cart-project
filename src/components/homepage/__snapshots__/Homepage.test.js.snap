// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomepageImage renders to match snapshot 1`] = `
<div>
  <div
    class="slideShowImage"
    style="background-image: url(mock-image1.jpg);"
  >
    <div
      class="overlay"
    />
    <div
      class="voucherAnnouncement"
    >
      <p
        class="announcement"
      >
        Use code 
        <span
          class="code"
        >
          mBright90
        </span>
         for 10% off your order
      </p>
    </div>
  </div>
</div>
`;
